src/main/java/tc/oc/pgm/rotation/MapPoll.java

```diff
@@ -193,14 +197,6 @@ public boolean toggleVote(MapInfo vote, UUID player) throws CommandException {
     return false;
   }
 
-  /** @return The map currently winning the vote, null if no vote is running. */
-  private MapInfo getMostVotedMap() {
```

Comment:
Don't get rid of this method, see comment about specific collection that caches vote count